WooCommerce/Classes/Extensions/UserDefaults+Woo.swift

```diff
@@ -94,3 +97,32 @@ extension UserDefaults {
         }
     }
 }
+
+// MARK: - Check for Blaze banner visibility
+extension UserDefaults {
+    /// Checks of the Blaze banner has been dismissed for a site.
+    /// - Parameters:
+    ///     - siteID: ID of the site to be checked for the Blaze banner visibility.
+    ///
+    func hasDismissedBlazeBanner(for siteID: Int64) -> Bool {
```

Comment:
Kindly consider adding separate unit tests for these two methods.

Reply:
Oh I missed this! I will add the unit tests in a separate PR, merging this for now.